Backpack/HorizontalNavigation/Classes/BPKHorizontalNavigation.m

```diff
@@ -192,7 +199,11 @@ - (void)setSize:(BPKHorizontalNavigationSize)size {
 #pragma mark - Private
 
 - (CGFloat)barHeight {
-    return BPKSpacingSm / 2;
+    return 3;
```

Comment:
Border sizes aren't yet available in `backpack-ios`, but could you add a comment that says this should be replaced with `BpkBorderSizeXl` when it's available?